.github/workflows/Test-documentation.yml

```diff
@@ -0,0 +1,17 @@
+name: Test documentation
+
+on: [ push, pull_request ]
+
+jobs:
+  tests:
+    name: Render documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Test if the documentation will render without warnings
+        run: |
+          mkdir -p Documentation-GENERATED-temp \
+          && docker run --rm --pull always -v $(pwd):/project \
+             ghcr.io/typo3-documentation/render-guides:latest --config=Documentation --no-progress --fail-on-log
```

Comment:
```suggestion
          make test-docs
```

Reply:
@mbrodala of coulrse we can change it what way you want. Everything is only a suggestions

Reply:
@linawolf There is a [composer script](https://github.com/TYPO3-Console/TYPO3-Console/blob/7916d393fe18b441d0e339fde6c9247fd49a5a58/composer.json#L118) which is not obsolete or outdated, right? And https://github.com/TYPO3-Console/TYPO3-Console/blob/7916d393fe18b441d0e339fde6c9247fd49a5a58/render-documentation.yml can also be removed?

Reply:
in general I would prefer to not add a Makefile, but update the one script already in composer.json and add a new one for the test